cl/recap/mergers.py

```diff
@@ -1734,8 +1734,17 @@ async def merge_attachment_page_data(
             try:
                 rd = await RECAPDocument.objects.aget(**params)
             except RECAPDocument.DoesNotExist:
-                rd = RECAPDocument(**params)
-                rds_created.append(rd)
+                try:
+                    del params["attachment_number"]
+                    del params["document_type"]
+                    params["pacer_doc_id"] = attachment["pacer_doc_id"]
+                    rd = await RECAPDocument.objects.aget(**params)
+                    if attachment.get("attachment_number", 0) != 0:
+                        rd.attachment_number = attachment["attachment_number"]
+                        rd.document_type = RECAPDocument.ATTACHMENT
+                except RECAPDocument.DoesNotExist:
+                    rd = RECAPDocument(**params)
```

Comment:
I think you'll need to make a copy of params up above before `del` keys from it or else this line won't have all the values it does now.

Reply:
Changed to deep copy params.